docs/features.rst

```diff
@@ -119,13 +119,13 @@ Recursive models
    >>> @dataclass
    ... class Order:
    ...     recipient: Person
-   ...     items: list[Item]
+   ...     items: List[Item]
```

Comment:
You also need to import `List` for this test. E.g. in the `testsetup` section on top of the file.